week-01/assessment/capsuleHotel/src/Hotel.java

```diff
@@ -0,0 +1,142 @@
+                    break;
+
+                case 3:
+                    view(console, guestList, capsulesCount);
+                    System.out.print("Select from the menu: [1. Check In, 2. Check Out, 3. View, and 4. Exit]");
+                    operation = console.nextInt();
+                    break;
+
+                case 4:
+                    exitConfirmation = exit(console);
+                    if (exitConfirmation) {
+                        break;
+                    }
+                    System.out.print("Select from the menu: [1. Check In, 2. Check Out, 3. View, and 4. Exit]");
+                    operation = console.nextInt();
+                    break;
+            }
+        }
+
+    }
+
+    //CHECK IN
+    public static boolean checkIn(Scanner console, String[] guestList) {
+        System.out.println("What is your name? ");
+        String name = console.next();
+
+        System.out.println("What is your capsule number? ");
+
+        int capsuleNumber = console.nextInt() - 1;
+        if (guestList[capsuleNumber] == null) {
+            guestList[capsuleNumber] = name;
+            return true;
+        } else {
+            System.out.println("This capsule is occupied. ");
+            return false;
+        }
+    }
+
+    //CHECK OUT
+
+    public static boolean checkOut(Scanner console, String[] guestList) {
+
+        System.out.println("What is your capsule number? ");
+        int capsuleNumber = console.nextInt() - 1;
+
+        if (guestList[capsuleNumber] == null) {
+            System.out.println("There is no one occupying that room. ");
+            return false;
+        } else {
+            guestList[capsuleNumber] = null;
+            System.out.println("Thank you for staying at the Capsule Hotel! You have successfully checked out. ");
+            return true;
+        }
+    }
+
+    //VIEW
+    public static void view(Scanner console, String[] guestList, int capsulesCount) {
+        System.out.println("What room do you to view? ");
+        int capsuleNumber = console.nextInt() - 1;
+        //Beginning case
+        if (capsuleNumber - 5 < 0) {
+            for (int j = 0; j < capsuleNumber + 6; j++) {
+                if (guestList[j] == null) {
+                    System.out.println("Room: " + (j + 1) + " is unoccupied");
+                } else {
+                    System.out.println("In room: " + (j + 1) + " is " + guestList[j]);
+                }
+            }
+        //Middle case
+        } else if (capsuleNumber + 6 < capsulesCount) {
+            for (int k = capsuleNumber - 5; k < capsuleNumber + 6; k++) {
+                if (guestList[k] == null) {
+                    System.out.println("Room: " + (k + 1) + " is unoccupied");
+                } else {
+                    System.out.println("In room: " + (k + 1) + " is " + guestList[k]);
+                }
+            }
+        //End case
+        } else if (capsuleNumber + 5 > capsulesCount) {
```

Comment:
could have also used the length property here 

```suggestion
        } else if (capsuleNumber + 5 > guestList.length) {
```